### PR TITLE
Correct path in example test command

### PR DESCRIPTION
**Pull request by mhw**
### Summary

The test files are in `test/view_component` now; this fixes the example to reflect that.

### CODE REVIEW

**Comment by Spone** [comment]
@mhw can you please add a changelog entry?

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ title: Changelog
 
 ## main
 
+* Correct path in example test command in Contributing document.
```
Comment (Spone) [inline docs/CHANGELOG.md:10]:
```suggestion
* Correct path in example test command in Contributing docs.
```

**Review by Spone — APPROVED** [review]

**Comment by Spone** [comment]
Thank you @mhw!

**Comment by Spone** [comment]
@BlakeWilliams any idea why the pvc job is failing repeatedly?

**Comment by BlakeWilliams** [comment]
@Spone Looks like a change upstream caused the tests to fail, but it was fixed in #1245.

**Pull request merged**